views/login.jsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import { Dimensions, ImageBackground, View } from 'react-native';
+import { Container, Header, Content, Card, CardItem, Thumbnail, Text, Button, Icon, Left, Label, Body, Right, Title, Form, Input, Item } from 'native-base';
+import { Font, AppLoading } from "expo";
+
+export default class LoginView extends React.Component {
+  static navigationOptions = { header: null };
+
+  constructor(props) {
+    super(props);
+    this.state = { 
+      loading: true,
+      registering: false,
+      username: null,
+      password: null,
+      firstName: null,
+      lastName: null,
+      goldenTicket: null
+    };
+  }
+
+  // This is just to avoid a bug with expo...
+  async componentWillMount() {
+    await Font.loadAsync({
+      Roboto: require("native-base/Fonts/Roboto.ttf"),
+      Roboto_medium: require("native-base/Fonts/Roboto_medium.ttf")
+    });
+    this.setState({ loading: false });
```

Comment:
This is some hacky workaround a bug with expo and can probably be removed but it was helpful for me while developing